src/index.js

```diff
@@ -4,6 +4,14 @@ export default [
   'Let’s debug it directly in production.',
   'Just ship it already.',
   'The leak is minor, we’ll just add more RAM later.',
-  'Looks like it’s all working fine',
-  'We’ll fix it later',
+  'Looks like it’s all working fine.',
+  'We’ll fix it later.',
+  'Does it support blockchain?',
+  "Didn't we refactor last year?",
+  'Technical debt? Never heard of it.',
+  'Writing tests takes x3 time than writing actual code.',
+  'We need you to work on features. Not performance.',
+  'Please do not worry a lot about quality. This is a prototype.',
+  'Hurry up. This is just a proof of concept.',
+  'Can we worry about security at a later phase?'
```

Comment:
@cherouvim suggestion to simplify: "Can we worry about security later?"

What do you think?

Reply:
Done.